.github/workflows/release.yml

```diff
@@ -33,3 +33,22 @@ jobs:
           body_path: release_description.md
           draft: false
           prerelease: false
+
+  npm-release:
+    needs: github-release
```

Comment:
Ahh, this line is what I'd overlooked. Didn't realize you'd already built in the workflow serialization. Nice!